sql/pre_install/tables.sql

```diff
@@ -477,6 +477,8 @@ CREATE TABLE _timescaledb_catalog.compression_settings (
   orderby text[],
   orderby_desc bool[],
   orderby_nullsfirst bool[],
+  minmax text[],
+  bloom text[],
```

Comment:
What's stored there, just the column names, and we create index per-column?

It's likely that we'll extend it soon:
* bloom filters get a parameter for false positive ratio, maybe for number of hashes as well
* we add new index types such as bloom filter of ngrams.

Maybe we should use array of text `key=value` pairs, like for reloptions? Like `array['type=bloom', 'column=device_id', 'false_positive=0.001']`. We can use all the standard Postgres functions like `parseRelOptions`.